{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -41,27 +41,27 @@ repos:
         types: [python]
 
   - repo: https://github.com/flakeheaven/flakeheaven
-    rev: 1.0.1
+    rev: 3.2.1
     hooks:
     -   id: flakeheaven
         name: flakeheaven
         exclude: .*migrations/.*
         args: [--config=pyproject.toml]
         additional_dependencies:
             - bandit==1.7.4
-            - flake8-bugbear==22.4.25
+            - flake8-bugbear==22.10.27
             - flake8-bandit==3.0.0
```

Comment:
There's a more recent update but it only supports newer flake8, while some other stuff in here requires slightly older flake8, so the easiest is to leave this at this version.

I think there's too many linters in here.